testgrid/config/README.md

```diff
@@ -164,7 +172,79 @@ test_groups:  # Attach to a specific test_group
     context_link: https://github.com/kubernetes/kubernetes/issues/123
 ```
 
+### What Counts as 'Recent'
+Configure `num_columns_recent` to change how many columns TestGrid should consider 'recent' for results.
+TestGrid uses this to calculate things like 'is this test stale?' (and hides the test).
+
+```
+test_groups:
+- name: kubernetes-build
+  gcs_prefix: kubernetes-jenkins/logs/ci-kubernetes-build
+  num_columns_recent: 3
+```
+
+## Using the client
+
+Here are some quick tips and clarifications for using the TestGrid site!
+
+### Customizing Test Result Sizes
+
+Change the size of the test result rectangles.
+
+The three sizes are Standard, Compact, and Super Compact. You can also specify
+`width=X` in the URL (X > 3) to customize the width. For small widths, this may
+mean the date and/or changelist, or other custom headers, are no longer
+visible.
```

Comment:
Noted; following up more thoroughly with screenshots for this later, filed #4040